packages/ui/components/dialog/Dialog.tsx

```diff
@@ -89,7 +89,7 @@ export const DialogContent = React.forwardRef<HTMLDivElement, DialogContentProps
         <DialogPrimitive.Content
           {...props}
           className={classNames(
-            "fadeIn bg-default scroll-bar fixed left-1/2 top-1/2 z-50 w-full max-w-[22rem] -translate-x-1/2 -translate-y-1/2 rounded-md text-left shadow-xl focus-visible:outline-none sm:align-middle",
```

Comment:
will removing w-full have side-effects @sean-brydon or @Udit-takkar ?

Reply:
Untill anyone of them reply, allow me to explain what w-full was doing generally..
w-full was allowing to take full width over every screen whether its small medium or large device.
Since dialog box is one if the child div that's why dislog was also taking full width irrespective of screen size.
In the similar manner, form component(which allow to overwrite date and has two divs, one to show date picker and one to show the selected dates) is child div of dialog and is taking full width, due to this right div (div to show dates selected) place is always visible even if dates are not selected.

Therefore to reduce size of form div, we have to fix the default size of the dialog and to fix the dialog we should fix its parent class.
Therefore what I did, I removed the default w-full class (which was by default for every device), and said if the screen is mobile then only take full width otherwise don't.

In this manner, the dialog will take the width if there is content to display, otherwise, it won't take any extra empty space.
This is how I came to fix it.

I hope it makes sense, by the way, I'm open to feedback and will fix this if needed again.

Reply:
lemme test this and merge asap! 🙏 thank you